src/components/tests/content-tab-snapshots.test.tsx

```diff
@@ -16,9 +16,10 @@ jest.mock( '../../hooks/use-offline' );
 
 jest.mock( '../../hooks/use-update-demo-site' );
 const updateDemoSiteMock = jest.fn();
+const isDemoSiteUpdating = jest.fn();
 ( useUpdateDemoSite as jest.Mock ).mockReturnValue( {
 	updateDemoSite: updateDemoSiteMock,
-	isDemoSiteUpdating: false,
+	isDemoSiteUpdating: isDemoSiteUpdating,
```

Comment:
Since the function `isDemoSiteUpdating` is not being used in the tests, what do you think about simply adding:
```
isDemoSiteUpdating: jest.fn(),
```

Reply:
I created an issue here to improve the tests: https://github.com/Automattic/dotcom-forge/issues/9008